cpu/esp32/periph/gpio.c

```diff
@@ -153,28 +153,29 @@ struct _rtc_gpio_t {
     uint8_t  mux;      /**< mux io/rtc bit [0..31] in the register, 32 - no mux */
     uint8_t  pullup;   /**< pullup bit [0..31] in the register, 32 - no pullup */
     uint8_t  pulldown; /**< pulldown bit [0..31] in the register, 32 - no pulldown */
+    uint8_t  drive;    /**< drive strength start bit [0..30] in the register */
```

Comment:
Does it make sense to have macros for all of this?

for example:

```c
    uint8_t  drive;    /**< drive strength start bit ESP_GPIO_DRIVE_TYPE in the register */
};

/* Table of RTCIO GPIO pins information */
static const struct _rtc_gpio_t _rtc_gpios[] = {
    {  0, RTC_IO_SENSOR_PADS_REG, 27, 32, 32, ESP_DRIVE_TYPE_SENSOR }, /* rtc0 (gpio36) SENSOR_VP/SENSE 1 */
 {  1, RTC_IO_SENSOR_PADS_REG, 26, 32, 32, ESP_DRIVE_TYPE_SENSOR }, /* rtc1 (gpio37) SENSOR_CAPP/SENSE 2 */

...
    {  9, RTC_IO_XTAL_32K_PAD_REG, 17, 22, 23, ESP_DRIVE_TYPE_XTAL_32K_P },/* rtc9 (gpio32) XTAL_32K_P */
    { 10, RTC_IO_TOUCH_PAD0_REG, 19, 27, 28, ESP_DRIVE_TYPE_TOUCH },  /* rtc10 (gpio4) TOUCH0 */
```
or do you think this will just create clutter and the numbers are sufficient?

Reply:
Since these number are just bit numbers, I don't think that macros would help.